src/styles/blocks/normalize.scss

```diff
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
```

Comment:
we can use only `box-sizing: border-box` for `*` selector.
If you need to reset margin for some tags you can use these tags as selectors